projects/Apps/common/src/index.ts

```diff
@@ -218,15 +218,7 @@ export const sizeDescriptions: { [k in ImageSize]: number } = {
     tabletXL: 1140,
 }
 
-export const Editions = [
-    'daily-edition',
-    'american-edition',
-    'australian-edition',
-    'training-edition',
-    'the-dummy-edition',
-] as const
-
-export type Edition = typeof Editions[number]
+export type Edition = string
```

Comment:
This seems to be used as an edition identifier, shall we rename it to `EditionId`?

Reply:
https://github.com/guardian/editions/pull/1499